beetsplug/fetchart.py

```diff
@@ -975,6 +974,26 @@ def get(self, album, plugin, paths):
             self._log.debug('Spotify: error loading response: {}'
                             .format(response.text))
             return
+
+
+class CoverArtUrl(RemoteArtSource):
+    NAME = "Cover Art URL"
+
+    def get(self, album, plugin, paths):
+        image_url = None
+        try:
+            image_url = album.items().get().cover_art_url
```

Comment:
I am somewhat surprised to see that this only looks at the item-level attribute, rather than an attribute on the album also. I presume this is because this is how metadata-source plugins want to provide this data, which makes sense. But perhaps we also want to check `album.cover_art_url` in case that changes in the future?

Reply:
Changed the logic to first look for `album.cover_art_url`